apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -36,12 +37,15 @@ const Auth: React.FC = () => {
 		return <SignInForm onSubmit={handleSignInSubmit} />;
 	};
 
-	return (
-		<>
-			state: {dataStatus}
-			{getScreen(pathname)}
-		</>
-	);
+	const handleRedirect = (): React.JSX.Element => {
+		if (dataStatus === DataStatus.FULFILLED) {
+			return <Navigate replace to={AppRoute.ROOT} />;
+		}
+
+		return <>{getScreen(pathname)}</>;
+	};
+
+	return <>{handleRedirect()}</>;
```

Comment:
Why do you need a separate function for that?
```suggestion
		if (dataStatus === DataStatus.FULFILLED) {
			return <Navigate replace to={AppRoute.ROOT} />;
		}

	return <>{getScreen(pathname)}</>;
```